policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/senior_or_disabled/assets/limit/couple.yaml

```
@@ -1,11 +1,11 @@
 description: Asset limit (of SSI countable assets) for a couple to qualify for Medicaid under the optional senior or disabled pathway.
 metadata:
   unit: currency-USD
-  label: "Medicaid senior or disabled asset limit (couple)"
+  label: Medicaid senior or disabled asset limit (couple).
```

```suggestion
  label: Medicaid senior or disabled asset limit (couple)
```
We usually dont include periods in labels